packages/storefront-lwc/src/modules/commerce/productTile/productTile.html

```diff
@@ -20,9 +20,10 @@
             <div class='price'>
                 <commerce-product-price product={product} context="tile"></commerce-product-price>
             </div>
-
-            <commerce-product-tile-swatches color-swatches={product.colorSwatches}></commerce-product-tile-swatches>
-
+            
+            <template if:true={product.colorSwatches}>
```

Comment:
If `product.colorSwatches` is an empty array then it will still display the product tile swatches or try to do we look for an empty array in the `product-tile-swatches` component?

Reply:
To display the product's recommendations, I didn't include colorSwatches in the typeDef, so product.colorSwatches will be false for all recommendations, so no color swatches will be displayed. For regular products, it will be as usual. do you think I need to add another check for empty array?

Reply:
I guess it doesn't really matter. The `getColorSwatches` function could return an empty array. The only benefit would be not rendering or going through the logic in the `product-tile-swatches` component when an empty array gets returned.